sdk/documenttranslation/Azure.AI.Translator.DocumentTranslation/README.md

```diff
@@ -514,18 +514,18 @@ When you submit a pull request, a CLA-bot will automatically determine whether y
 
 This project has adopted the [Microsoft Open Source Code of Conduct][code_of_conduct]. For more information see the [Code of Conduct FAQ][coc_faq] or contact [opencode@microsoft.com][coc_contact] with any additional questions or comments.
 
-![Impressions](https://azure-sdk-impressions.azurewebsites.net/api/impressions/azure-sdk-for-net%2Fsdk%2Fdocumenttranslation%2FAzure.AI.DocumentTranslation%2FREADME.png)
+![Impressions](https://azure-sdk-impressions.azurewebsites.net/api/impressions/azure-sdk-for-net%2Fsdk%2Fdocumenttranslation%2FAzure.AI.Translator.DocumentTranslation%2FREADME.png)
 
 
 <!-- LINKS -->
-[documenttranslation_client_src]: https://github.com/Azure/azure-sdk-for-net/tree/master/sdk/documenttranslation/Azure.AI.DocumentTranslation/src
+[documenttranslation_client_src]: https://github.com/Azure/azure-sdk-for-net/tree/master/sdk/documenttranslation/Azure.AI.Translator.DocumentTranslation/src
```

Comment:
You'll probably need to update the links in another PR. But if you keep the original links, it will break ci. 🤔

Reply:
I thought the CI was going to fail with this update, but looking at the CI it actually passed https://dev.azure.com/azure-sdk/public/_build/results?buildId=818670&view=logs&jobId=340cbc18-e772-5986-7674-8efce120e265&j=b70e5e73-bbb6-5567-0939-8415943fadb9&t=bdeefc16-b669-5ebd-ad94-a2c19ade53b0
so I think it should be ok